### DISCUSSION TITLE
docker-compose always failed after setup-buildx-action

### DISCUSSION THREAD

**Discussion by Kang-Hou** [docker/setup-buildx-action#350] [General] [answered]
My action firstly setup-buildx-action and then run `docker-compose up -d`. It used to work well but this failed a lot of times this week and says 'docker-compose' command not found.

Is there any recent changes introduce the problem?

**Comment by crazy-max** [comment, marked as answer]
`docker-compose` aka compose v1 is EOL for quite some time. Looking at github public runners only compose v2 is available: https://github.com/actions/runner-images/blob/main/images%2Fubuntu%2FUbuntu2204-Readme.md

You should change your command to `docker compose` which uses compose v2 as CLI plugin.

**Reply by Kang-Hou** [reply]
This really helps! Thanks
